examples/nextjs-rsc/app/page.tsx

```diff
@@ -0,0 +1,20 @@
+const Page = async () => {
+  return (
+    <>
+      <h1>Next.js 13 + Grafbase</h1>
+      <p>
+        Here you can select a post from the menu on the left and open it inside
+        of a nested route.
+      </p>
+      <h3>You will need to</h3>
+      <ol>
+        <li>
+          Go to <a href="http://localhost:4000">http://localhost:4000</a>
```

Comment:
Use relative path here so it works in prod?

Reply:
We’d need to switch to use a NEXT_PUBLIC environment variable. I can do this.

Reply:
I've updated the text here to be more generic @fbjork